static/docs/command-reference/checkout.md

```diff
@@ -21,34 +21,31 @@ which instance of each data file or directory should be used, with the checksums
 saved in the `outs` field. The `dvc checkout` command updates the workspace data
 to match with the <abbr>cached</abbr> files that correspond to those checksums.
 
-Using an SCM like Git, the DVC-files are kept under version control. At a given
-branch or tag of the SCM repository, the DVC-files will contain checksums for
-the corresponding data files kept in the cache. After an SCM command like
-`git checkout` is run, the DVC-files will change to the state at the specified
-branch or commit or tag. Afterwards, the `dvc checkout` command is required in
-order to synchronize the data files with the currently checked out DVC-files.
+When using Git, the DVC-files can be kept under version control. DVC-files in
+different branches or tags may contain checksums for different data files, saved
+by DVC in the project's cache. The DVC-files in the workspace, or the value of
+their `outs` fields can change when using `git checkout`. The `dvc checkout`
+command is required in this situation, in order to synchronize the data files
+(tracked by DVC) with the checked out DVC-files.
 
-This command must be executed after `git checkout` since Git doesn't track files
-that are under DVC control. For convenience a Git hook is available, simply by
-running `dvc install`, that will automate running `dvc checkout` after
-`git checkout`. See `dvc install` for more information.
+For convenience a Git hook is available to automate running `dvc checkout` after
+`git checkout`. To install it, use `dvc install`.
 
 The execution of `dvc checkout` does:
 
-- Scan the `outs` entries in DVC-files to compare with the currently checked out
-  data files. The scanned DVC-files is limited by the listed `targets` (if any)
-  on the command line. And if the `--with-deps` option is specified, it scans
-  backward from the given `targets` in the corresponding
-  [pipeline](/doc/command-reference/pipeline).
+- Scan the `outs` entries in DVC-files to compare with the <abbr>outputs</abbr>
```

Comment:
And again, not sure why are we changing this.

Reply:
The only changes in this bullet were to introduce the "outputs" tooltip, and remove the note about `--with-deps` which I think is unnecessary (since this rare option is explained later in the **Options** section). I do realize now maybe "outputs" should be introduced earlier in the description so it makes more sense to use the term here. Just did that in 56f0780a.

Reply:
I think this is resolved.